nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/service/ColdStorageServiceImpl.java

```diff
@@ -235,22 +240,21 @@ public DocumentModel proceedMoveToColdStorage(CoreSession session, DocumentRef d
         if (session.isUnderRetentionOrLegalHold(documentRef)) {
             log.debug("The document {} is under retention or legal hold and cannot be moved to cold storage",
                     () -> documentRef);
-            throw new NuxeoException(String.format(
+            throw new DocumentSecurityException(String.format(
                     "The document %s is under retention or legal hold and cannot be moved to cold storage",
-                    documentRef), SC_FORBIDDEN);
+                    documentRef));
         }
         if (!session.hasPermission(documentRef, SecurityConstants.WRITE_COLD_STORAGE)) {
             log.debug("The user {} does not have the right permissions to move the content of document {}",
                     session::getPrincipal, () -> documentModel);
-            throw new NuxeoException(String.format("The document: %s cannot be moved to cold storage", documentRef),
-                    SC_FORBIDDEN);
+            throw new DocumentSecurityException(
+                    String.format("The document: %s cannot be moved to cold storage", documentRef));
```

Comment:
Maybe "User %s is not authorized to move doc: %s to cold storage" or "Unauthorized move to cold storage for doc: %s"